library/src/main/java/com/novoda/downloadmanager/NotificationInformation.java

```diff
@@ -2,7 +2,7 @@
 
 import android.app.Notification;
 
-public interface NotificationInformation {
+interface NotificationInformation {
```

Comment:
This class can be converted to a data class quite easily now but the pull request was already big enough.